docs/editors.md

```diff
@@ -40,7 +40,7 @@ Another option is to use [yajs.vim](https://github.com/othree/yajs.vim) with
 
 ### Visual Studio Code
 
-Install the [Babel ES6/ES7](https://marketplace.visualstudio.com/items?itemName=dzannotti.vscode-babel-coloring)
```

Comment:
funny that it's called Sublime Babel - should we just be more specific and add `sublime-babel-vscode` as the title?